lottie/src/main/java/com/airbnb/lottie/Layer.java

```diff
@@ -244,30 +244,24 @@ static Layer newInstance(JSONObject json, LottieComposition composition) {
         preCompHeight = (int) (json.optInt("h") * composition.getScale());
       }
 
-      long inFrame = json.optLong("ip");
-      long outFrame = json.optLong("op");
+      float inFrame = json.optLong("ip");
```

Comment:
Hmm, I don't think these can be float values. It's getting a long from the json anyways.
Can you change the json to `optInt` and all of the vars to int and Integer?